autoload/codefmt/buildifier.vim

```diff
@@ -39,7 +39,11 @@ function! codefmt#buildifier#GetFormatter() abort
   " @flag(buildifier)
   " @throws ShellError
   function l:formatter.Format() abort
+    let l:lint_flag = s:plugin.Flag('buildifier_lint_mode')
     let l:cmd = [ s:plugin.Flag('buildifier_executable') ]
+    if l:lint_flag != ""
+      let l:cmd += ["--lint=" .. l:lint_flag]
```

Comment:
Is this `..` a typo intended to be `.`? AFAICT it blows up with an error on my version of vim.

Reply:
How strange! I'm using NeoVim and always used two dots to concatenate strings, I never knew one dot sufficed. It's fixed now.